src/sql-parser/src/ast/defs/statement.rs

```diff
@@ -3586,12 +3586,18 @@ impl<T: AstInfo> AstDisplay for RefreshOptionValue<T> {
     }
 }
 
-#[derive(Debug, Clone, PartialEq, Eq, Hash, PartialOrd, Ord)]
+#[derive(Debug, Clone, PartialEq, Eq, Hash, PartialOrd, Ord, Deserialize, Serialize)]
 pub enum ClusterScheduleOptionValue {
     Manual,
     Refresh,
 }
 
+impl Default for ClusterScheduleOptionValue {
+    fn default() -> Self {
+        ClusterScheduleOptionValue::Manual
+    }
+}
```

Comment:
As a general rule we don't impl `Default` for AST types, because they are not meant to be constructed directly, and the "defaults" flow from parser.rs and the SQL planner. Does it cause any friction to leave `Default` unimplemented here, and declare the default in the `generate_extracted_config!` macro as was done previously?

Reply:
Oh, I see, `ALTER RESET`. Ok, never mind.